contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/rule_based_profiler/domain_builder/data_profiler_column_domain_builder.py

```diff
@@ -110,18 +103,37 @@ def _get_domains(
         # Obtain profile_path from "rule state" (i.e., variables and parameters); from instance variable otherwise.
         profile_path: str = get_parameter_value_and_validate_return_type(
             domain=None,
-            parameter_reference=self.profile_path,
+            parameter_reference=f"{VARIABLES_KEY}profile_path",
             expected_return_type=str,
             variables=variables,
             parameters=None,
         )
 
+        metric: str = get_parameter_value_and_validate_return_type(
+            domain=None,
+            parameter_reference=f"{VARIABLES_KEY}metric",
+            expected_return_type=str,
+            variables=variables,
+            parameters=None,
+        )
+
+        metric_values: str = get_parameter_value_and_validate_return_type(
```

Comment:
@alexjeilert The name `metric_values` is not ideal here; I believe these are the parameters of the metric you are going to use, so I would suggest a more fitting name (whereas "metric value" usually refers to the value of a metric that was computed -- this appears to be different).  Thank you.